lib/chelsea/cli.rb

```diff
@@ -73,7 +73,7 @@ def _submit_sbom(gems)
       
       return unless status_url
 
-      iq.poll_status(status_url)
+      exit iq.poll_status(status_url)
```

Comment:
Was it not exiting with the code? Previously this would have been an implicit return (which is ruby-ish)

Reply:
Thanks, didn't realize it would do that without being explicit (I'm ruby clueless).

My later machinations change this code some more, but I'll look into being more ruby-ish. (That looks awfully close to "rubbish",  so maybe I go with "ruby-esque" ;)